public/app/features/query/state/PanelQueryRunner.ts

```diff
@@ -189,12 +191,12 @@ export class PanelQueryRunner {
     }
   }
 
-  private pipeToSubject(observable: Observable<PanelData>) {
+  private pipeToSubject(observable: Observable<PanelData>, panelId?: number) {
     if (this.subscription) {
       this.subscription.unsubscribe();
     }
 
-    this.subscription = observable.subscribe({
+    this.subscription = mergePanelAndDashData(observable, getDashboardQueryRunner().getResult(panelId)).subscribe({
```

Comment:
Here we need to figure out which panel plugins that should subscribe to DashboardQueryRunner results, for now all PanelRunners subscribe.

Reply:
Fixed by #33253